heudiconv/heuristics/reproin.py

```diff
@@ -483,7 +483,6 @@ def infotodict(seqinfo):
             # 3 - Image IOD specific specialization (optional)
             dcm_image_iod_spec = s.image_type[2]
             image_type_seqtype = {
-                'P': 'fmap',   # phase
```

Comment:
if you could split out typo fixes to a separate commit/PR, I would be happy to accept it really quickly ;)